src/resources/views/fields/browse_multiple.blade.php

```diff
@@ -0,0 +1,174 @@
+@php
+$multiple = array_get($field, 'multiple', true);
+$value = old($field['name']) ? old($field['name']) : (isset($field['value']) ? $field['value'] : (isset($field['default']) ? $field['default'] : null));
```

Comment:
That ternary is somewhat...unreadable.

Reply:
That ternary is copied from the browse field (and I have seen it in others too), so if this needs changing, then maybe a (general) overhaul of the way the value is determined is in order.

Reply:
Yeah, fair enough.